Lib/asyncio/base_events.py

```diff
@@ -812,7 +812,7 @@ def call_at(self, when, callback, *args, context=None):
         timer = events.TimerHandle(when, callback, args, self, context)
         if timer._source_traceback:
             del timer._source_traceback[-1]
-        heapq.heappush(self._scheduled, timer)
+        heapq.heappush(self._scheduled, (when, timer))
```

Comment:
```suggestion
        # The `TimerHandle` is wrapped in a tuple to avoid calling the
        # `TimerHandle.__lt__` method since the `heapq` operations
        # will need to compare against many other `TimerHandler` object.
        heapq.heappush(self._scheduled, (when, timer))
```